test/parallel/test-whatwg-url-domainto.js

```diff
@@ -35,6 +36,13 @@ const tests = require('../fixtures/url-idna.js');
 }
 
 {
+  for (const [i, { ascii, unicode }] of testsHyphenDomains.valid.entries()) {
+    assert.strictEqual(ascii, domainToASCII(unicode),
```

Comment:
Oh..never mind, it is supposed to check uppercase characters will be converted to lowercase ones...(is there any in the test cases?)